apps/ff-site/src/app/ecosystem-explorer/utils/getEcosystemProjectData.ts

```diff
@@ -38,7 +38,7 @@ function transformEcosystemProjectData(
     ...project,
     seo: {
       ...project.seo,
-      title: project.seo.title || `${project.title} ${METADATA_TITLE_SUFFIX}`,
+      title: project.seo.title || `${project.title}${METADATA_TITLE_SUFFIX}`,
```

Comment:
Super nit, but I quite like how this reads when it's just two consts.

```suggestion
      title: project.seo.title || project.title + METADATA_TITLE_SUFFIX,
```